schema/gen/go/genBoolReprBool.go

```diff
@@ -77,7 +77,7 @@ func (g boolReprBoolReprGenerator) EmitNodePrototypeType(w io.Writer) {
 	`, w, g.AdjCfg, g)
 }
 func (g boolReprBoolReprGenerator) GetNodeBuilderGenerator() NodeBuilderGenerator {
-	return boolReprBoolReprBuilderGenerator{g.AdjCfg, g.Type}
```

Comment:
I don't know if there are implications for this switch, performance or data modifiability but staticcheck (S1016) really doesn't like the current pattern

Reply:
it looks like this has the same effect